src/cdogs/music.c

```diff
@@ -133,7 +133,11 @@ void MusicPause(SoundDevice *device)
 {
 	if (device->musicStatus == MUSIC_PLAYING)
 	{
-		Mix_PauseMusic();
+		// fade out music to finish 1 second from now
+		while(!Mix_FadeOutMusic(1000) && Mix_PlayingMusic()) {
+			// wait for any fades to complete
+			SDL_Delay(100);
```

Comment:
I don't think MusicPause is currently used, but this is called when the game is in progress, it would freeze while the music is fading out. This will affect network multiplayer games and frame skipping.